test/functional/api/nvim_spec.lua

```diff
@@ -0,0 +1,100 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
this would go in `test/functional/lua/utility_functions_spec.lua` or some other `test/functional/lua/*.lua` file.